--- schedtool-1.0/Makefile.orig	2004-04-06 23:47:53.000000000 +0200
+++ schedtool-1.0/Makefile	2004-04-21 20:11:43.131205040 +0200
@@ -1,16 +1,18 @@
 # go on and adjust here if you don't like those flags
-CFLAGS=-Os -fomit-frame-pointer -s -pipe
+CFLAGS=$(OPTFLAGS) -pipe
 #CFLAGS=-Wall -Os -fomit-frame-pointer -s -pipe -DDEBUG
 CC=gcc
 # likewise, if you want to change the destination prefix
-DESTPREFIX=/usr/local
-DESTDIR=bin
+PREFIX=/usr
+BINDIR=$(PREFIX)/bin
+MANDIR=$(PREFIX)/share/man
+DESTDIR=
 GZIP=gzip -9
 TARGET=schedtool
 DOCS=README INSTALL SCHED_DESIGN TUNING
 RELEASE=$(shell basename `pwd`)
 
-all: affinity zipman
+all: affinity
 
 clean:
 	rm -f *.o $(TARGET)
@@ -19,10 +21,10 @@
 	rm -f *~ *.s
 
-install: all install-doc
-	install -d $(DESTPREFIX)/$(DESTDIR)
-	install -c $(TARGET) $(DESTPREFIX)/$(DESTDIR)
-	install -d $(DESTPREFIX)/man/man8
-	install -c schedtool.8.gz $(DESTPREFIX)/man/man8
+install: all
+	install -d $(DESTDIR)$(BINDIR)
+	install -c $(TARGET) $(DESTDIR)$(BINDIR)
+	install -d $(DESTDIR)$(MANDIR)/man8
+	install -c schedtool.8 $(DESTDIR)$(MANDIR)/man8
 
 install-doc:
 	install -d $(DESTPREFIX)/share/doc/schedtool
