--- schedtool-1.2.4/Makefile.orig	2004-11-28 12:17:27.000000000 +0100
+++ schedtool-1.2.4/Makefile	2004-11-29 23:01:12.434669960 +0100
@@ -1,10 +1,12 @@
 # go on and adjust here if you don't like those flags
-CFLAGS=-Os -fomit-frame-pointer -s -pipe
+CFLAGS=$(OPTFLAGS) -pipe
 #CFLAGS=-Wall -Os -fomit-frame-pointer -s -pipe -DDEBUG
 CC=gcc
 # likewise, if you want to change the destination prefix
-DESTPREFIX=/usr/local
-DESTDIR=bin
+PREFIX=/usr
+BINDIR=$(PREFIX)/bin
+MANDIR=$(PREFIX)/share/man
+DESTDIR=
 GZIP=gzip -9
 TARGET=schedtool
 DOCS=README INSTALL SCHED_DESIGN TUNING
@@ -18,11 +20,11 @@
 distclean: clean unzipman
 	rm -f *~ *.s
 
-install: all install-doc zipman
-	install -d $(DESTPREFIX)/$(DESTDIR)
-	install -c $(TARGET) $(DESTPREFIX)/$(DESTDIR)
-	install -d $(DESTPREFIX)/man/man8
-	install -c schedtool.8.gz $(DESTPREFIX)/man/man8
+install: all
+	install -d $(DESTDIR)/$(BINDIR)
+	install -c $(TARGET) $(DESTDIR)/$(BINDIR)
+	install -d $(DESTDIR)$(MANDIR)/man8
+	install -c schedtool.8 $(DESTDIR)$(MANDIR)/man8
 
 install-doc:
 	install -d $(DESTPREFIX)/share/doc/schedtool
